src/nav-bar/index.js

```diff
@@ -67,19 +67,27 @@ export default createComponent({
           style={{ zIndex: this.zIndex }}
           class={[bem({ fixed: this.fixed }), { [BORDER_BOTTOM]: this.border }]}
         >
-          <div class={bem('left')} onClick={this.onClickLeft}>
+          {this.hasLeft && <div class={bem('left')} onClick={this.onClickLeft}>
             {this.genLeft()}
-          </div>
+          </div>}
           <div class={[bem('title'), 'van-ellipsis']}>
             {this.slots('title') || this.title}
           </div>
-          <div class={bem('right')} onClick={this.onClickRight}>
+          {this.hasRight && <div class={bem('right')} onClick={this.onClickRight}>
             {this.genRight()}
-          </div>
+          </div>}
         </div>
       );
     },
 
+    hasLeft() {
+      return this.leftArrow || this.leftText || this.leftSlot;
```

Comment:
`this.leftSlot` does not exist

Reply:
Oops, I prepared a fixup but didn't push it before creating the PR.